packages/controller/src/resources/cortex/index.ts

```diff
@@ -327,7 +327,7 @@ export function CortexResources(
   let serviceAccountName: string | undefined = undefined;
   if (target === "gcp") {
     annotations = {
-      "iam.gke.io/gcp-service-account": gcp!.cortexServiceAccount
+      "iam.gke.io/gcp-service-account": gcp?.cortexServiceAccount
```

Comment:
We should explicitly assert that this exists before we rely on it with `assert(gcp?.cortexServiceAccount)`